openstack/baremetal/v1/nodes/requests.go

```diff
@@ -636,3 +636,21 @@ func SetRAIDConfig(client *gophercloud.ServiceClient, id string, raidConfigOptsB
 	_, r.Header, r.Err = gophercloud.ParseResponse(resp, err)
 	return
 }
+
+// Get the current BIOS Settings for the given Node.
+func ListBIOSSettings(client *gophercloud.ServiceClient, id string) (r BIOSSettingsResult) {
+	resp, err := client.Get(biosListSettingsURL(client, id), &r.Body, &gophercloud.RequestOpts{
+		OkCodes: []int{200},
+	})
+	_, r.Header, r.Err = gophercloud.ParseResponse(resp, err)
+	return
+}
+
+// Get one BIOS Setting for the given Node.
+func GetSingleBIOSSetting(client *gophercloud.ServiceClient, id string, setting string) (r SingleBIOSSettingResult) {
```

Comment:
Total nit pick: How about `GetBIOSSetting`? The use of "Get" (instead of "List") and the singular use of "Setting" implies that a single item will be returned. This is not a required change, though.

Reply:
Yes, that naming is more clear. Fixed.